### PR TITLE
python311Packages.llm: 0.14 -> 0.15, python311Packages.sqlite-utils: 3.36 -> 3.37

### PR DESCRIPTION
**Pull request by pimeys**
## Description of changes

Updated the package llm from 0.14 to 0.15. It requires a new version of sqlite-utils, which is also updated in this PR. The package name has changed to use underscore in PyPi.

https://llm.datasette.io/en/stable/changelog.html

https://github.com/simonw/sqlite-utils/releases/tag/3.37

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

ping @aldoborrero

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/development/python-modules/sqlite-utils/default.nix [hunk 1]
```diff
@@ -14,17 +14,17 @@
   testers,
   sqlite-utils,
 }:
-
 buildPythonPackage rec {
   pname = "sqlite-utils";
-  version = "3.36";
+  version = "3.37";
   format = "setuptools";
```
Comment (dotlambda) [inline pkgs/development/python-modules/sqlite-utils/default.nix:20]:
```suggestion
  pyproject = true;
```
and add `build-system = [ setuptools ]`

File: pkgs/development/python-modules/sqlite-utils/default.nix [hunk 2]
```diff
@@ -14,17 +14,17 @@
...
 
   src = fetchPypi {
-    inherit pname version;
-    hash = "sha256-3MMROU/obcFvZQN7AHXiOO/P0uEuZdU+0ZaVRQKZbzw=";
+    inherit version;
+    pname = "sqlite_utils";
+    hash = "sha256-VCpxAz1OeTb+kJIwrJeU0+IAAhg4q2Pbrzzo9bwic6Q=";
   };
 
   postPatch = ''
```
Comment (dotlambda) [inline pkgs/development/python-modules/sqlite-utils/default.nix:30]: can be removed

**Review by dotlambda — COMMENTED** [review]
Please replace `nativeBuildInputs` with `build-system` and `propagatedBuildInputs` with `dependencies`.

**Comment by techknowlogick** [comment]
Result of `nixpkgs-review pr 333251` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>16 packages built:</summary>
  <ul>
    <li>github-to-sqlite (python312Packages.github-to-sqlite)</li>
    <li>github-to-sqlite.dist (python312Packages.github-to-sqlite.dist)</li>
    <li>llm (python312Packages.llm)</li>
    <li>llm.dist (python312Packages.llm.dist)</li>
    <li>python311Packages.github-to-sqlite</li>
    <li>python311Packages.github-to-sqlite.dist</li>
    <li>python311Packages.llm</li>
    <li>python311Packages.llm.dist</li>
    <li>python311Packages.sqlite-migrate</li>
    <li>python311Packages.sqlite-migrate.dist</li>
    <li>python311Packages.sqlite-utils</li>
    <li>python311Packages.sqlite-utils.dist</li>
    <li>python312Packages.sqlite-migrate</li>
    <li>python312Packages.sqlite-migrate.dist</li>
    <li>sqlite-utils (python312Packages.sqlite-utils)</li>
    <li>sqlite-utils.dist (python312Packages.sqlite-utils.dist)</li>
  </ul>
</details>

**Comment by dotlambda** [comment]
> Please replace `nativeBuildInputs` with `build-system` and `propagatedBuildInputs` with `dependencies`.

Still

**Comment by pimeys** [comment]
> > Please replace `nativeBuildInputs` with `build-system` and `propagatedBuildInputs` with `dependencies`.
> 
> Still

should be correct now

**Review by techknowlogick — APPROVED** [review]
thanks. with @dotlambda's requested changes sqlite-utils still works as expected. I'll leave the llm testing to others tho.

**Pull request merged**